src/Pages/ContactPage.jsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Container,
+  Typography,
+  Box,
+  Paper,
+  makeStyles,
+} from "@material-ui/core";
+import Footer from "../Components/Footer";
+
+const useStyles = makeStyles({
+  bg: {
+    backgroundColor: "var(--light-gray)",
```

Comment:
I like CSS vars lol. Could we do the same for other colors throughout the PR? This could be instead of doing the constants.js approach I mentioned above.

Reply:
Yes for sure!